packages/cherry-pick-bot/src/bot.ts

```diff
@@ -240,8 +250,55 @@ export = (app: Probot) => {
         repo,
         [context.payload.pull_request.merge_commit_sha],
         targetBranch,
+        remoteConfig.preservePullRequestTitle
+          ? `${context.payload.pull_request.title} (cherry-pick #${context.payload.pull_request.number})`
+          : undefined,
         logger
       );
     }
   });
+
+  app.on(
+    [
+      'pull_request.opened',
+      'pull_request.reopened',
+      'pull_request.edited',
+      'pull_request.synchronize',
+    ],
+    async context => {
+      let octokit: Octokit;
+      if (context.payload.installation?.id) {
+        octokit = await getAuthenticatedOctokit(
+          context.payload.installation.id
+        );
+      } else {
+        throw new Error(
+          'Installation ID not provided in pull_request event.' +
+            ' We cannot authenticate Octokit.'
+        );
+      }
+
+      const logger = getContextLogger(context);
+      const {owner, repo} = context.repo();
+      const repoUrl = context.payload.repository.full_name;
+
+      // We should first check the config schema. Otherwise, we'll miss
+      // the opportunity for checking the schema when adding the config
+      // file for the first time.
+      const configChecker = new ConfigChecker<Configuration>(
+        schema,
+        CONFIGURATION_FILE_PATH
+      );
+      const valid = await configChecker.validateConfigChanges(
+        octokit,
+        owner,
+        repo,
+        context.payload.pull_request.head.sha,
+        context.payload.pull_request.number
+      );
+      if (!valid) {
+        logger.info(`cherry-pick-bot config is not valid for ${repoUrl}.`);
```

Comment:
This happens against a pull request. If we want to log this, maybe add pr_number too.

Reply:
Added `pr_number` to the log.